src/GitHub.VisualStudio/UI/Views/GitHubConnectContent.xaml.cs

```diff
@@ -101,7 +101,7 @@ public object Convert(object[] values, Type targetType, object parameter, Cultur
 
             var item = values[0] as ISimpleRepositoryModel;
             var context = values[1] as IGitAwareItem;
-            return context?.ActiveRepoUri == item?.CloneUrl && context?.ActiveRepo?.LocalPath == item?.LocalPath;
+            return context?.ActiveRepoUri == item?.CloneUrl && String.Equals(context?.ActiveRepo?.LocalPath, item?.LocalPath, StringComparison.OrdinalIgnoreCase);
```

Comment:
So, something that just occurred to me. Is `context?.ActiveRepoUri` a `UriString`? Is `item?.CloneUrl` too? If either is a string, then we'll get a string comparison here because of the implicit conversion of `UriString` to `string`. Even if they're both `UriString`, I'm not sure our `Equals` implementation does the right thing here. :frowning: Something worth testing out.